tests/libYARP_OS/PortTest.cpp

```diff
@@ -1582,6 +1582,16 @@ class PortTest : public UnitTest {
         checkTrue(p.close(),"Closing the broken_device");
     }
 
+    void testPortResumeDeadlock() {
+        report(0, "testing lockup if resume is called when not interrupted");
+        Port p;
+        p.open("/test");
+        p.interrupt();
+        p.resume();
+        p.resume(); //should not lockup here
+        checkTrue(p.close(), "port closed successfully");
```

Comment:
`p.close()` return void, not a bool it should be like this.
```suggestion
        p.close();
```
if you want to check the compilation of tests in `ccmake` enable `YARP_COMPILE_TESTS`